### PR TITLE
GRW-2055/Update nav component

### PR DESCRIPTION
**Pull request by amandaflod**
## Describe your changes

Update the nav component to match the current designs (at least a little bit better 😅). The Desktop design for the product list is still to be fixed, it just wouldn't collaborate with me today so maybe someone else can give it a go with fresh eyes!

https://user-images.githubusercontent.com/59652803/214590098-8fd44f91-aead-4059-a8a6-1b4692a00c4e.mov

https://user-images.githubusercontent.com/59652803/214590277-9f5a827c-6e51-47f1-a468-60fd9b5c899f.mov

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

[GRW-2055]: https://hedvig.atlassian.net/browse/GRW-2055?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by robinandeer — COMMENTED** [review]
Just noticed that the menu disappears when moving the cursor from the menu down to the popover because there is some space between them 🤔 

Perhaps we can just add some transparent div there to prevent it -- but doable in a follow-up PR

File: apps/store/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "../../node_modules/typescript/lib",
```
Comment (robinandeer) [inline apps/store/.vscode/settings.json:2]: did you mean to add this file?

File: apps/store/public/mockServiceWorker.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (0.47.3).
+ * Mock Service Worker (0.44.2).
```
Comment (robinandeer) [inline apps/store/public/mockServiceWorker.js:5]: this seems downgraded - must be a mistake, right? Maybe you just need to run "yarn install"

File: apps/store/src/blocks/HeaderBlock.tsx [hunk 3]
```diff
@@ -65,6 +74,47 @@ export const NestedNavContainerBlock = ({ blok }: NestedNavContainerBlockProps)
...
+  buttons: ExpectedBlockType<ButtonBlockProps>
+  currentActiveItem?: string
+}>
+
+export const ProductNavContainerBlock = ({ blok }: ProductNavContainerBlockProps) => {
+  const filteredNavItems = filterByBlockType(blok.navItems, NavItemBlock.blockName)
+  const buttonBlocks = filterByBlockType(blok.buttons, ButtonBlock.blockName)
+
+  return (
+    <NavigationMenuPrimitiveItem value="hello" {...storyblokEditable(blok)}>
```
Comment (robinandeer) [inline apps/store/src/blocks/HeaderBlock.tsx:89]: oh we should change this back -- value should not be "hello" hehe

File: apps/store/src/components/Header/Header.tsx [hunk 4]
```diff
@@ -1,35 +1,55 @@
...
 
 export const Wrapper = styled.header(({ theme }) => ({
   display: 'flex',
-  justifyContent: 'space-between',
   alignItems: 'center',
+  justifyContent: 'space-between',
   width: '100%',
   height: MENU_BAR_HEIGHT_MOBILE,
-  padding: theme.space[4],
+  padding: ` 0 ${theme.space[4]}`,
```
Comment (robinandeer) [inline apps/store/src/components/Header/Header.tsx:13]: nit: use semantic space names

File: apps/store/src/components/Header/TopMenuMobile/TopMenuMobile.tsx [hunk 5]
```diff
@@ -49,22 +58,27 @@ export const TopMenuMobile = ({ children }: TopMenuMobileProps) => {
...
+              <ButtonWrapper x={0.25}>
+                <Button variant="secondary" size="medium">
+                  <StyledAppleIcon />
+                  App Store
+                </Button>
+                <Button variant="secondary" size="medium">
+                  <StyledAndroidIcon />
+                  Google Play
+                </Button>
+              </ButtonWrapper>
```
Comment (robinandeer) [inline apps/store/src/components/Header/TopMenuMobile/TopMenuMobile.tsx:79]: these buttons don't seem to do anything -- guess they should be links but we can fix later

File: packages/ui/src/components/HedvigLogo/HedvigLogo.tsx [hunk 6]
```diff
@@ -1,14 +1,18 @@
...
-    <g>
-      <path d="M89.077 4.815a4.198 4.198 0 00-4.199 4.199 4.198 4.198 0 004.2 4.198 4.198 4.198 0 004.198-4.198 4.198 4.198 0 00-4.199-4.2zm0 7.818a3.624 3.624 0 01-3.619-3.62 3.624 3.624 0 013.62-3.618 3.624 3.624 0 013.618 3.619 3.624 3.624 0 01-3.619 3.619z" />
-      <path d="M90.33 8.712h-2.482V6.717h-.58v4.57h.58V9.292h2.482v1.995h.557v-4.57h-.557z" />
-    </g>
+export const HedvigLogo = ({
+  color = 'currentColor',
+  width = 78,
```
Comment (robinandeer) [inline packages/ui/src/components/HedvigLogo/HedvigLogo.tsx:9]: is this safe to change -- have you checked if we use this somewhere already without setting width/height?

File: apps/store/src/components/Header/TopMenuMobile/TopMenuMobile.tsx (hunk 5 above)
Comment (amandaflod) [reply apps/store/src/components/Header/TopMenuMobile/TopMenuMobile.tsx:79]: Yes I have put that down as a todo :))

File: packages/ui/src/components/HedvigLogo/HedvigLogo.tsx (hunk 6 above)
Comment (amandaflod) [reply packages/ui/src/components/HedvigLogo/HedvigLogo.tsx:9]: I have checked! :)

File: apps/store/src/components/Header/Header.tsx (hunk 4 above)
Comment (amandaflod) [reply apps/store/src/components/Header/Header.tsx:13]: Will fix!

File: apps/store/.vscode/settings.json (hunk 1 above)
Comment (amandaflod) [reply apps/store/.vscode/settings.json:2]: No, will look into this :/

File: apps/store/public/mockServiceWorker.js (hunk 2 above)
Comment (amandaflod) [reply apps/store/public/mockServiceWorker.js:5]: Does not help unfortunately :/ But will revert the changes!

**Comment by amandaflod** [comment]
Here is my todo-list for the remaining stuff! Will create tickets for these 😇

Screenshot 2023-01-25 at 16 51 41

**Review by robinandeer — APPROVED** [review]
Let's ship it!

**Pull request merged**